### PR TITLE
Revert "use known-working Windows OS image"

### PR DESCRIPTION
**Pull request by marosset**
This reverts commit d535a5d2cbd7da42bf6591c8d005594d0df53485.

 

 
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: templates/test/ci/prow-ci-version/patches/machine-deployment-ci-version.yaml [hunk 1]
```diff
@@ -12,4 +12,4 @@ spec:
           publisher: cncf-upstream
           offer: capi
           sku: ubuntu-1804-gen1
-          version: latest
+          version: "latest"
```
Comment (mboersma) [inline templates/test/ci/prow-ci-version/patches/machine-deployment-ci-version.yaml:15]: Did this need to be quoted?

Comment (marosset) [reply templates/test/ci/prow-ci-version/patches/machine-deployment-ci-version.yaml:15]: I'm not sure, should I update this?

Comment (mboersma) [reply templates/test/ci/prow-ci-version/patches/machine-deployment-ci-version.yaml:15]: <shrug> No I was just curious, it seems like a no-op change and now I see where it comes from (it was at "latest" with quotes before #2616 merged).

**Pull request merged**